bwproject.py

```diff
@@ -59,7 +59,7 @@ def _test_auth(self, username, token):
         if "username" in user:
             if username is None:
                 return user["username"], token
-            elif user["username"] == username:
+            elif user["username"].lower() == username.lower():
```

Comment:
Keep reading the word `username`... make jess go crazy. 
Is there a reason we shouldn't just assign the variable `username` to `username.lower()` towards the beginning?

Reply:
Yeh that would work too but this feels more definite and it reflects how the API backend works: Store usernames in mixed case but always compare case insensitively.